packages/dev/package.json

```diff
@@ -57,6 +57,7 @@
     "@netlify/config": "^23.0.4",
     "@netlify/dev-utils": "2.2.0",
     "@netlify/functions": "3.1.9",
+    "@netlify/headers": "^0.0.0",
```

Comment:
I think we want to pin an exact version here.

Reply:
I'll make that change for internal consistency, but what are we solving for here and how do we plan to enforce it?

Reply:
When referencing packages from the monorepo, we always know the exact version we want to use. We're not enforcing that anywhere.